infrastructure/loadtesting/terraform/docker/loadtest.Dockerfile

```
@@ -1,6 +1,6 @@
-FROM golang:1.19.12@sha:ddbb6cbbc88a5e8d802c3ab7dc717d7a0634401c030266f4ff0f1933806f2ed9
+FROM golang:1.21.1@sha:0dff643e5bf836005eea93ad89e084a17681173e54dbaa9ec307fd776acab36e
```

Note that this has to be `sha256`:
```suggestion
FROM golang:1.21.1@sha256:0dff643e5bf836005eea93ad89e084a17681173e54dbaa9ec307fd776acab36e
```